src/app/rpmostree-builtin-upgrade.c

```diff
@@ -60,8 +60,7 @@ rpmostree_builtin_upgrade (int             argc,
   g_autoptr(GOptionContext) context = g_option_context_new ("");
   glnx_unref_object RPMOSTreeOS *os_proxy = NULL;
   glnx_unref_object RPMOSTreeSysroot *sysroot_proxy = NULL;
-  g_autoptr(GVariant) previous_default_deployment = NULL;
-  g_autoptr(GVariant) new_default_deployment = NULL;
+  g_autoptr(GVariant) default_deployment = NULL;
```

Comment:
Let's still call this `new_default_deployment`?  Otherwise it seems confusing, since there clearly *is* a default deployment, we just didn't get a signal.